without.js

```diff
@@ -0,0 +1,5 @@
+function without(array, val) {
+    array = array.filter(item => !val.includes(item));
+    return array;
```

Comment:
`return array.filter(item => !val.includes(item));`

Reply:
or `const without = (array, val) => array.filter(item => !val.includes(item));`

Reply:
fixed in 00dc018